### PR TITLE
Hardware Access Control and Sonata Software Book

### PR DESCRIPTION
**Pull request by HU90m**

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
Thanks for walking me through these changes. Approving so that we can get this in and iterate over it later. I'll do a more detailed review and run through later and propose changes in a follow up PR if necessary.

File: exercises/hardware_access_control/README.md
Comment (johngt) [inline exercises/hardware_access_control/README.md:156]: I'd suggest at this point adding a reference to joining the Technical Interest Group - approach one of us for joining that.
Also suggest that there is the Technology Access Programme - TAP - Cohort 6 that lowRISC is helping to adjudicate if people are interested in taking the demos further.
They should also reach out to us via info@lowrisc.org if they have collaboration ideas.

If people have issues they can reach us for the next month on ches-2024@lowrisc.org

**Review by johngt — COMMENTED** [review]
Some changes suggested

File: exercises/hardware_access_control/README.md
Comment (AlexJones0) [inline exercises/hardware_access_control/README.md:139]: > which adds an addition condition

Typo: Which adds an **additional** condition.

File: doc/exploring-cheriot-rtos.md
Comment (AlexJones0) [inline doc/exploring-cheriot-rtos.md:14]: > When starting first starting to explore capabilities

Typo: When ~starting~ first starting to explore capabilities

File: doc/exploring-cheriot-rtos.md
Comment (AlexJones0) [inline doc/exploring-cheriot-rtos.md:11]: > ... and add functionality can be found in ...

and **additional** ~add~ functionality.

File: doc/guide/running-software.md
Comment (AlexJones0) [reply doc/guide/running-software.md:39]: This is not necessarily true; for example in my Linux environment it is under `/media/$USER/SONATA` instead. I think it should either retain this wording or needs to be more specific

File: exercises/hardware_access_control/README.md
Comment (AlexJones0) [inline exercises/hardware_access_control/README.md:42]: > The keen eyed among you will ...

Grammar: The **keen-eyed** among you will ...

File: exercises/hardware_access_control/README.md
Comment (AlexJones0) [inline exercises/hardware_access_control/README.md:87]: > These come from superfluous lines in `led_walk_dynamic.cc`, that releases ownership of user LED 3 ...`

Grammar: These come from superfluous lines in `led_walk_dynamic.cc`, **which release** ownership of user LED 3 ...

("lines" is plural, "that releases" is third-person singular)

**Comment by HU90m** [comment]
Thank you very much @AlexJones0 and @johngt , these are great spots.

Currently blocked on https://github.com/lowRISC/sonata-software/pull/34 , before I can make the changes.

File: doc/guide/running-software.md
Comment (marnovandermaas) [reply doc/guide/running-software.md:39]: I would leave it as "on my computer" as well.

File: doc/exploring-cheriot-rtos.md
Comment (marnovandermaas) [reply doc/exploring-cheriot-rtos.md:4]: Yes, here: https://github.com/CHERIoT-Platform/book
But we don't have to link that here right?

File: doc/exploring-cheriot-rtos.md
Comment (marnovandermaas) [reply doc/exploring-cheriot-rtos.md:42]: Already done.

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:11]: Already done.

File: README.md
Comment (marnovandermaas) [reply README.md:28]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:7]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:73]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:156]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:139]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: doc/exploring-cheriot-rtos.md
Comment (marnovandermaas) [reply doc/exploring-cheriot-rtos.md:14]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: doc/exploring-cheriot-rtos.md
Comment (marnovandermaas) [reply doc/exploring-cheriot-rtos.md:11]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:42]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

File: exercises/hardware_access_control/README.md
Comment (marnovandermaas) [reply exercises/hardware_access_control/README.md:87]: Addressed in: https://github.com/lowRISC/sonata-software/pull/42

**Pull request merged**
